examples/.tailcallrc.graphql

```diff
@@ -10,7 +10,14 @@ directive @server(
   vars: [KeyValue]
   responseHeaders: [KeyValue]
   hostname: String
+  version: HttpVersion
 ) on SCHEMA
+
+enum HttpVersion {
+  HTTP1
+  HTTP2
```

Comment:
Add a Fallback mode which will cork on http1 and try upgrade if receives a Http2 request